modules/media/jsx/mediaIndex.js

```diff
@@ -90,18 +154,23 @@ class MediaIndex extends React.Component {
             id="media_filter_form"
             ref="mediaFilter"
             columns={3}
-            formElements={this.state.Data.form}
+            formElements={this.state.data.form}
             onUpdate={this.updateFilter}
             filter={this.state.filter}
           >
             <br/>
-            <ButtonElement label="Clear Filters" type="reset" onUserInput={this.resetFilters}/>
+            <ButtonElement
+              label="Clear Filters"
+              type="reset"
+              onUserInput={this.resetFilters}
+            />
           </FilterForm>
           <StaticDataTable
-            Data={this.state.Data.Data}
-            Headers={this.state.Data.Headers}
+            Data={this.state.data.Data}
+            Headers={this.state.data.Headers}
+            hiddenHeaders={this.state.hiddenHeaders}
```

Comment:
i just realized that this.props.hiddenHeaders hasn't been implemented yet on minor

Reply:
Nope... hence the frustration of these sort of in-between PRs while waiting for the merge. Can remove them or leave them in the meantime but would really not like to keep going back and forth on them.